### DISCUSSION TITLE
Past, present and future of rust-protobuf

### DISCUSSION THREAD

**Discussion by stepancheg** [stepancheg/rust-protobuf#606] [Announcements]
Just want to share the post again. [HERE](https://github.com/stepancheg/rust-protobuf/blob/master/doc/past-present-future.md)

**Comment by winksaville** [comment]
@stepancheg, I read the the "PPF of rust-protobuf" and felt you were saying you didn't have time for rust-protobuf. But now I see you've put significant effort recently. Have you changed your mind on rust-protobuf's future? 

Is V3 support going to be released?

**Reply by stepancheg** [reply]
> Is V3 support going to be released?

Yes, soon. I'm polishing the API now (and feedback would be appreciated).

**Reply by stepancheg** [reply]
> Have you changed your mind on rust-protobuf's future?

There future is still not very clear, but I've got request from some people to release V3, so I decided to do the release.

**Reply by winksaville** [reply]
I wish I could help, but I know nothing about V3 :(
